src/SFA.DAS.ProviderCommitments/SFA.DAS.ProviderCommitments.Web/Validators/AddDraftApprenticeshipViewModelValidator.cs

```diff
@@ -10,6 +10,7 @@ public AddDraftApprenticeshipViewModelValidator()
             RuleFor(x => x.DateOfBirth).Must(y => y.IsValid).WithMessage("The Date of birth is not valid").When(z => z.DateOfBirth.HasValue);
             RuleFor(x => x.StartDate).Must(y => y.IsValid).WithMessage("The start date is not valid").When(z => z.StartDate.HasValue);
             RuleFor(x => x.EndDate).Must(y => y.IsValid).WithMessage("The end date is not valid").When(z => z.EndDate.HasValue);
+            RuleFor(x => x.ActualStartDate).Must(y => y.IsValid).WithMessage("The start date is not valid").When(z => z.ActualStartDate.HasValue);
```

Comment:
AC 6 in the story reads like this must be valid and non-optional and therefore the ".When" should not be there, perhaps I am missing something though and maybe this is just because it is a draft?